pkgs/development/ocaml-modules/reason-native/cli.nix

```diff
@@ -1,7 +1,12 @@
-{ re, reason, pastel, ... }:
+{ lib, buildDunePackage, re, reason, pastel, src }:
+
+buildDunePackage {
+  inherit src;
 
-{
   pname = "cli";
+  version = "0.0.1-alpha-unstable-2024-05-07";
+
+  duneVersion = "3";
```

Comment:
I think you can remove all these `duneVersion` attributes.

Reply:
Right you are. From `pkgs/build-support/ocaml/dune.nix`:

```
let Dune =
  let dune-version = args.duneVersion or "3"; in
  { "1" = dune_1; "2" = dune_2; "3" = dune_3; }."${dune-version}"
; in
```